content/en/docs/collector/internal-telemetry.md

```diff
@@ -113,3 +118,148 @@ journalctl | grep otelcol | grep Error
+
+- Total received data, broken down by receivers.
+
+- Total exported data, broken down by exporters.
+
+- Total dropped data due to throttling, broken down by data type.
+
+- Total dropped data due to invalid data received, broken down by data type.
+
+- Total incoming connection count, broken down by receiver.
+
+- Uptime since start.
+
+### List of internal metrics
+
+The following table identifies each internal metric by name and description.
+Each metric is also categorized by instrumentation type and level of verbosity.
+
+| Metric name                                       | Description                                                                               | Type      | Level      |
+| ------------------------------------------------- | ----------------------------------------------------------------------------------------- | --------- | ---------- |
+| `otelcol_exporter_enqueue_failed_log_records`     | Number of spans that exporter(s) failed to enqueue.                                       | Counter   |            |
+| `otelcol_exporter_enqueue_failed_metric_points`   | Number of metric points that exporter(s) failed to enqueue.                               | Counter   |            |
+| `otelcol_exporter_enqueue_failed_spans`           | Number of spans that exporter(s) failed to enqueue.                                       | Counter   |            |
+| `otelcol_exporter_queue_capacity`                 | Fixed capacity of the retry queue, in batches.                                            | Gauge     |            |
+| `otelcol_exporter_queue_size`                     | Current size of the retry queue, in batches.                                              | Gauge     |            |
+| `otelcol_exporter_send_failed_log_records`        | Number of logs that exporter(s) failed to send to destination.                            | Counter   |            |
+| `otelcol_exporter_send_failed_metric_points`      | Number of metric points that exporter(s) failed to send to destination.                   | Counter   |            |
+| `otelcol_exporter_send_failed_spans`              | Number of spans that exporter(s) failed to send to destination.                           | Counter   |            |
+| `otelcol_exporter_sent_log_records`               | Number of logs successfully sent to destination.                                          | Counter   |            |
+| `otelcol_exporter_sent_metric_points`             | Number of metric points successfully sent to destination.                                 | Counter   |            |
+| `otelcol_exporter_sent_spans`                     | Number of spans successfully sent to destination.                                         | Counter   |            |
+| `otelcol_http_client_active_requests`             | Number of active HTTP client requests.                                                    | Counter   | `detailed` |
```

Comment:
Tagging @dmitryax and @mx-psi for this question. I understand that the default level for internal metrics was changed from `basic` to `normal` in https://github.com/open-telemetry/opentelemetry-collector/pull/9767. Does that mean none of these metrics now have a default level of basic? And if so, for my own understanding, why were the `processor_batch` metrics separately updated to `normal`? I'd like to fill in the rest of the Level column, but I'm not sure whether to put `basic` or `normal`. Please advise. Thanks!

Reply:
I spent some time working through this question on my own, and I've found the answer concerning the `processor_batch` metrics and the default level. (I misunderstood how the default level was being applied.) 

However, short of learning Go :smile:, I am really, truly stuck on how to find the level for specific metrics in the code. If @dmitryax or one of the other @open-telemetry/collector-approvers could let me know if any other* metrics are emitted at a level other than `basic`, I would be very grateful. Thank you!

*Other than:
`normal`:
`otelcol_processor_batch_batch_send_size`  
`otelcol_processor_batch_batch_size_trigger_send`  
`otelcol_processor_batch_metadata_cardinality`  
`otelcol_processor_batch_timeout_trigger_send`

`detailed`:
`otelcol_http_client_active_requests`  
`otelcol_http_client_connection_duration`  
`otelcol_http_client_open_connections`  
`otelcol_http_client_request_body_size`  
`otelcol_http_client_request_duration`  
`otelcol_http_client_response_body_size`  
`otelcol_http_server_active_requests`  
`otelcol_http_server_request_body_size`  
`otelcol_http_server_request_duration`  
`otelcol_http_server_response_body_size`  
`otelcol_rpc_client_duration`  
`otelcol_rpc_client_request_size`  
`otelcol_rpc_client_requests_per_rpc`  
`otelcol_rpc_client_response_size`  
`otelcol_rpc_client_responses_per_rpc`  
`otelcol_rpc_server_duration`  
`otelcol_rpc_server_request_size`  
`otelcol_rpc_server_requests_per_rpc`  
`otelcol_rpc_server_response_size`  
`otelcol_rpc_server_responses_per_rpc`

Reply:
@tiffany76 this is the correct understanding. All the metrics other than those listed above are emitted in the `basic` level. `normal` is the level applied by default. It means that metrics with `basic` (most of the metrics) and `normal` (`otelcol_processor_batch_*` metrics) are emitted by default, while `otelcol_http_*` and `otelcol_rpc_*`are not. BTW the `otelcol_` prefix was recently dropped for them. Now they are emitted as `http_*` and `rpc_*`.

Reply:
@tiffany76 - will you be adding `basic` to the **Level** column value for the relevant entries in the table? That would be my preference (and I think it's yours too based on an earlier comment).

Reply:
Table is complete now. Resolving this conversation.